<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgaddfood</class>
 <widget class="QDialog" name="dlgaddfood">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="cbType">
       <item>
        <property name="text">
         <string>主食</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>饮品</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>炒菜</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>靓汤</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>价格：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lePrice"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>元</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pbAdd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QPushButton" name="pbExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
